### ISSUE TITLE
Triple Frontier trailer

### ISSUE THREAD

**Issue by erpadmin** [andrewbrey/netflix-tweaked#18]
Firefox latest.
4 rows past the billboard there is a Triple Frontier "larger" listing that plays on mouse over.   other than the billboard and individual/smaller listings dont play as expected

**Comment by andrewbrey** [comment]
@erpadmin So, since first starting this extension, I've learned that there are lots of different versions of Netflix that are served to users. This is (apparently) influenced by lots of things from what a user's watch history is, to which A-B tests Netflix is currently doing on their UI.

What you're describing is a trailer that I've never encountered so that's going to make it very hard to _efficiently_ handle in the extension code. I have a round of updates I plan to make to the extension in which I more or less intend to take the nuclear option as far as identifying and preventing trailers, but I haven't found the time to get to it. If you're interested in taking a look at it yourself and making a pull request, you are very welcome to do so - in fact I encourage it!

Just a warning though, you will want to disable the automatic reload features of the development toolchain because if you refresh Netflix a bunch while you're developing, Netflix will temporarily suspend your account ☹ (I learned this the hard way...more than once lol).
